syllabus/3-Blockchain/2-P2P_Networks_State_Machines-slides.md

```diff
@@ -101,43 +122,147 @@ This led to Metallica filing a lawsuit against Napster for copyright infringemen
 Napster had to comply by banning hundreds of thousands of users from their platform who were sharing Metallica's music.
 This was a turning point in digital copyright law and played a significant role in Napster's eventual shutdown in 2001.
 
----
+---v
 
 ## Napster Setup
 
 <img style="width: 600px" src="./img/napster_1.svg" />
 
----
+---v
 
 ## Napster Setup
 
 <img style="width: 600px" src="./img/napster_2.svg" />
 
----
+---v
 
 ## Napster Setup
 
 <img style="width: 600px" src="./img/napster_3.svg" />
 
----
+---v
 
 ## Napster Setup
 
 <img style="width: 600px" src="./img/napster_4.svg" />
 
 ---
 
-## Gnutella(Limewire)
+## Limewire
+
+Free Software p2p file sharing **client**
+
+<!-- .element: class="fragment" data-fragment-index="2" -->
+
+In its history, it has mainly used two protocols
+
+<!-- .element: class="fragment" data-fragment-index="3" -->
+
```

Comment:
```suggestion

Notes:

First Gnutella, and then later BitTorrent. Let's look at each of them.
```

I think it is wise to mention the two protocols immediately, before you cover them individually. Otherwise, by the time you finish Gnutella and get on to BitTorrent, people might not even remember that they are in the middle of a list of two protocols used by lime wire.